### PR TITLE
#164798284 Deploy and setup base application to Heroku

### PR DESCRIPTION
**Pull request by Genza999**
**What does this PR do?**

- Sets up the application onto the Heroku server

- Creates pipelines for the application

**Description of tasks to be completed**

- Add procfile to the project's root directory

- Install gunicorn and django-heroku dependencies

- Update requirements.txt

- Configure settings.py file for Heroku setup

- Add an app.json file to the root directory

**How should this be manually tested?**

The application can be accessed through the applications Heroku base url [here](https://ah-backend-prime-staging.herokuapp.com/admin/login/?next=/admin/)

**Any background context you want to provide?**
The application uses pipelines that enable us to have automatic deployments for all pull requests. The pipeline has three stages i.e. Review apps, staging and production. Review apps creates deployments for our PR's, staging for our develop branch and production for master branch.

**What are the relevant pivotal tracker stories?**
[#164798284](https://www.pivotaltracker.com/story/show/164798284)

**Screenshots:**

**Heroku deployment**

Screen Shot 2019-03-28 at 00 55 03

**Pipelines**

Screen Shot 2019-03-28 at 09 45 57

### CODE REVIEW

**Review by patrickf949 — APPROVED** [review]
Great work @Genza999

**Review by anyatibrian — APPROVED** [review]
great work , i liked the way you have implement it

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python3.7"
+}
```
Comment (e-ian) [inline .vscode/settings.json]: @Genza999 please donot push .vscode/settings. Add to gitignore

Comment (Genza999) [reply .vscode/settings.json]: Thanks. Done!

Comment (e-ian) [reply .vscode/settings.json]: LGTM

**Review by e-ian — APPROVED** [review]

**Review by inno-asiimwe — APPROVED** [review]

**Pull request merged**